Documentation/Usage.md

```diff
@@ -143,84 +143,86 @@ To `import Clibgit`, the package manager requires that the libgit2 library has
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+**Note:** the system library may be located elsewhere on your system, such as
+`/usr/local/`, or `/usr/`. The paths above are valid for Homebrew on macOS.
 
-Note that the system library may be located elsewhere on your system, such as
-`/usr/` rather than `/usr/local/`.
-
-Create a directory called `Clibgit` next to the `example` directory and
-initialize it as a package that builds a system module:
-
-    example$ cd ..
-    $ mkdir Clibgit
-    $ cd Clibgit
-    Clibgit$ swift package init --type empty
-
-This creates a `Package.swift` file in the directory.
-Edit `Package.swift` and add `pkgConfig` parameter:
+Now let's define `Clibgit` dependency in your `Package.swift` in the example project:
 
 ```swift
-// swift-tools-version:5.3
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        // Targets are the basic building blocks of a package, defining a module or a test suite.
+        // Targets can depend on other targets in this package and products from dependencies.
+        .executableTarget(
+            name: "example",
+
+            // example executable requires "Clibgit" target as it's dependency.
+            // It's a systemLibrary target defined below.
+            dependencies: ["Clibgit"],
+            path: "Sources"
+        ),
+
+        // systemLibrary is a special type of build target that wraps a system library
+        // in a target that other targets can require as their depencency.
+        .systemLibrary(
+            name: "Clibgit",
+            pkgConfig: "libgit2",
+            providers: [
+                .brew(["libgit2"]),
+                .apt(["libgit2-dev"])
+            ]
+        )
+    ]
 )
+
 ```
 
 The `pkgConfig` parameter helps SwiftPM in figuring out the include and library
-search paths for the system library. Note: If you don't want to use the `pkgConfig`
+search paths for the system library. You can look up pkg-config for a package
+by running the following:
+
+    example$ pkg-config --cflags libgit2
+    -I/opt/homebrew/Cellar/libgit2/1.6.4/include
+
+**Note:** If you don't want to use the `pkgConfig`
 parameter you can pass the path of a directory containing the library using the
 `-L` flag in commandline when building your app:
 
-    example$ swift build -Xlinker -L/usr/local/lib/
+    example$ swift build -Xlinker -L/opt/homebrew/lib/
 
-Create a `module.modulemap` file so it consists of the following:
+Next, we'll create a directory `Sources/Clibgit` in your `example` projcet, and
+add a `module.modulemap` to it:
 
     module Clibgit [system] {
-      header "/usr/local/include/git2.h"
+      header "/opt/homebrew/include/git2.h"
```

Comment:
ditto re. `/usr/local` with note that homebrew users this may be `/opt/homebrew/`